From aefbbb813cdc27252153d2cf043900681c8689d8 Mon Sep 17 00:00:00 2001
From: Samuel Hym <samuel@tarides.com>
Date: Fri, 5 Jul 2024 12:58:58 +0200
Subject: [PATCH 06/14] Strip tmpheader.exe also on Windows

GNU strip can be called safely on binaries generated by cl as well as by
MinGW GCC (even if it doesn't produce a smaller executable for
cl-generated binaries) so invoke strip also on Windows so that MinGW
binaries are properly stripped
Tested with GNU strip 2.42
---
 stdlib/Makefile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/stdlib/Makefile b/stdlib/Makefile
index b6286920f2..a7f2e60a7a 100644
--- a/stdlib/Makefile
+++ b/stdlib/Makefile
@@ -96,10 +96,7 @@ endif
 .INTERMEDIATE: tmpheader.exe
 tmpheader.exe: $(HEADERPROGRAM).$(O)
 	$(V_MKEXE)$(call MKEXE_VIA_CC,$@,$^)
-# FIXME This is wrong - mingw could invoke strip; MSVC equivalent?
-ifneq "$(UNIX_OR_WIN32)" "win32"
 	$(STRIP) $@
-endif
 
 stdlib.cma: $(OBJS)
 	$(V_LINKC)$(CAMLC) -a -o $@ $^
-- 
2.45.2

